src/Website/Controllers/ContactUsController.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+using AttributeRouting.Web.Mvc;
+
+namespace mbsoft.BrewClub.Website.Controllers
+{
+    public class ContactUsController : Controller
+    {
+        [GET("ContactUs")]
```

Comment:
I'm fine with going all lowercase if that's a standard, otherwise it strikes me as a bit odd to abandon camel or pascal casing which provide greater readability than something that is all of one casing type be it all upper or all lowercase.